apps/trackers/tests/test_file_offer.py

```diff
@@ -0,0 +1,314 @@
+"""
+Bugzilla specific tracker test cases
```

Comment:
I guess that the module description is just copy-pasted and not updated.

Reply:
Updated for correct description.